poetry.lock

```diff
@@ -74,13 +74,13 @@ test = ["tox"]
 
 [[package]]
 name = "babel"
-version = "2.15.0"
+version = "2.16.0"
```

Comment:
Presumable this update was not intentional? It would have to be done in a separate PR on its own